Src/Screens/DashboardScreen/Viewables/ExpandableRow.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ExpandableRow.swift
+//  ugglan
+//
+//  Created by Axel Backlund on 2019-03-29.
+//
+
+import Flow
+import Form
+import Foundation
+import UIKit
+
+struct ExpandableRow<Content: Viewable, ExpandableContent: Viewable> where
+    Content.Matter == UIView,
```

Comment:
This should probably be a Generic that extends UIView, so:

```suggestion
    Content.Matter == ContentMatter,
```